docs/root/api-v2/config/wasm/wasm.rst

```diff
@@ -4,5 +4,3 @@ WASM
 .. toctree::
   :glob:
   :maxdepth: 2
-
```

Comment:
Let's remove the entire file then

Reply:
Ah yes, +1, thanks.

/wait

Reply:
I think this file still needs deleting?

/wait